addons/website_quote/models/sale_quote.py

```diff
@@ -55,21 +55,32 @@ def open_template(self):
 class SaleQuoteLine(models.Model):
     _name = "sale.quote.line"
     _description = "Quotation Template Lines"
-    _order = 'sequence, id'
+    _order = 'quote_id, sequence, id'
```

Comment:
Is it really needed to change the order ?

Reply:
Since the order is very important for sections to work correctly, it's better to be safe. We wouldn't want to mix up sections from different quotes in random places.